app/models/intake.rb

```diff
@@ -333,13 +344,19 @@ def self.searchable_fields
     dependents.with_deleted.each(&:destroy!)
   end
 
-  attr_encrypted :primary_last_four_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  attr_encrypted :spouse_last_four_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  attr_encrypted :primary_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  attr_encrypted :spouse_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  attr_encrypted :bank_name, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  attr_encrypted :bank_routing_number, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  attr_encrypted :bank_account_number, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
+  attr_encrypted :attr_encrypted_primary_last_four_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }, attribute: 'encrypted_primary_last_four_ssn'
+  attr_encrypted :attr_encrypted_spouse_last_four_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }, attribute: 'encrypted_spouse_last_four_ssn'
+  attr_encrypted :attr_encrypted_primary_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }, attribute: 'encrypted_primary_ssn'
+  attr_encrypted :attr_encrypted_spouse_ssn, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }, attribute: 'encrypted_spouse_ssn'
+  attr_encrypted :attr_encrypted_bank_name, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }, attribute: 'encrypted_bank_name'
```

Comment:
I'm confused with this line because I thought **bank name** is not an encrypted attribute

Reply:
These attributes are the _existing_ attribtues, getting prefixed with attr_encrypted so that we can do the data transition. We're writing the data onto new fields. For 7/9, they'll be written onto new encrypted fields. For bank_name and bank_routing_number, we're going to forgo encrypting them and just store them raw since they're not particularly sensitive or private on their own.